state/postgres_server_stats.go

```diff
@@ -0,0 +1,7 @@
+package state
+
+// PostgresServerStats - Statistics for a Postgres server.
+type PostgresServerStats struct {
+	CurrentXactId   Xid
+	NextMultiXactId Xid
```

Comment:
Both of these should be 64-bit types (we could invent a new Xid8 type for use here, to match Postgres)

Reply:
Or maybe Xid32 for the existing Xid, and Xid64 for this here?